### PR TITLE
Isotope find

### PR DESCRIPTION
**Pull request by jbae11**
Addresses and
solves #20

### CODE REVIEW

**Comment by jbae11** [comment]
@pep8speaks  pep8ify

**Comment by andrewryh** [comment]
Tested?

File: saltproc/core [hunk 1]
```diff
@@ -0,0 +1,105 @@
+set title "themperature coefficient of reactivity 900K for both"
```
Comment (katyhuff) [inline saltproc/core]: Does this need to be included? should it be called core or should it have a more meaningful name/location?

File: saltproc/fuel_comp_025U [hunk 2]
```diff
@@ -0,0 +1,7 @@
+mat fuel    -3.266364 burn 1 rgb 253 231 37
```
Comment (katyhuff) [inline saltproc/fuel_comp_025U]: I feel this shouldn't be added to the toplevel directory. Perhaps there is  a more organized place to place it.

File: saltproc/fuel_comp_with_fix [hunk 3]
```diff
@@ -0,0 +1,1159 @@
+mat  fuel  7.77767011499957E-02 fix 10c 1000 burn 1 vol 4.87100E+07 % density 3.266364g/cm3 for 1000K
```
Comment (katyhuff) [inline saltproc/fuel_comp_with_fix]: I feel this shouldn't be added to the top level directory. Perhaps there is a more organized place to place it.

File: saltproc/run_saltproc.py [hunk 4]
```diff
@@ -11,40 +11,49 @@
 from pyne import serpent
 from pyne import nucname
 
+##############################################################
+######### THIS IS THE INPUT FILE SECTION OF SALTPROC #########
+######### TO RUN, simply set these settings and run  #########
```
Comment (katyhuff) [inline saltproc/run_saltproc.py]: Comments like this don't generally make their way to the user. It's much better in the readme. Also, a great (and testable) way to make this appear on the command line is to use if __name__ == '__main__' and argparse with a help flag instead.

File: saltproc/run_saltproc.py [hunk 5]
```diff
@@ -11,40 +11,49 @@
...
-restart = 'False'
+
+# executable path of Serpent
+exec_path = '/projects/sciteam/bahg/serpent30/src/sss2'
+
+# Number of cores and nodes to use in cluster
 cores = 32
-# Parse flags
-# Read run command
-parser = argparse.ArgumentParser()
```
Comment (katyhuff) [inline saltproc/run_saltproc.py]: why did we get rid of argparse here? Argparse is great.

File: saltproc/whole_core_axial_ref [hunk 6]
```diff
@@ -0,0 +1,726 @@
+% --- Surfaces (core pitch = 10.16 cm): Zone I-A
```
Comment (katyhuff) [inline saltproc/whole_core_axial_ref]: does this need to be in the top level directory? Is there a better place to keep these?

**Comment by jbae11** [comment]
@katyhuff  Thanks for your review! I reflected your comments

**Comment by andrewryh** [comment]
Tested.
Looks good for me

**Pull request merged**